src/main/docs/guide/httpServer/sessions.adoc

```diff
@@ -1,131 +1 @@
-
-You can then pass this `AUTHORIZATION_INFO` in subsequent requests to reuse the existing link:{micronautsessionapi}/io/micronaut/session/Session.html[Session]:
-
-.Sending the AUTHORIZATION_INFO header
-snippet::io.micronaut.docs.session.ShoppingControllerSpec[tags="add", indent=0]
-
-<1> The `AUTHORIZATION_INFO` is retrieved from the response
-<2> And then sent as a header in the subsequent request
-
-== Using @SessionValue
-
-Rather than explicitly injecting the link:{micronautsessionapi}/io/micronaut/session/Session.html[Session] into a controller method, you can instead use link:{micronautsessionapi}/io/micronaut/session/annotation/SessionValue.html[@SessionValue]. For example:
-
-snippet::io.micronaut.docs.session.ShoppingController[tags="view", indent=0,title="Using @SessionValue"]
-
-<1> link:{micronautsessionapi}/io/micronaut/session/annotation/SessionValue.html[@SessionValue] is declared on the method resulting in the return value being stored in the link:{micronautsessionapi}/io/micronaut/session/Session.html[Session]. Note that you must specify the attribute name when used on a return value
-<2> link:{micronautsessionapi}/io/micronaut/session/annotation/SessionValue.html[@SessionValue] is used on a `@Nullable` parameter which results in looking up the value from the link:{micronautsessionapi}/io/micronaut/session/Session.html[Session] in a non-blocking way and supplying it if present. In the case a value is not specified to link:{micronautsessionapi}/io/micronaut/session/annotation/SessionValue.html[@SessionValue] resulting in the parameter name being used to lookup the attribute.
-
-== Session Events
-
-You can register api:context.event.ApplicationEventListener[] beans to listen for link:{micronautsessionapi}/io/micronaut/session/Session.html[Session] related events located in the link:{micronautsessionapi}/io/micronaut/session/event/package-summary.html[session.event] package.
-
-The following table summarizes the events:
-
-.Session Events
-|===
-|Type|Description
-
-|link:{micronautsessionapi}/io/micronaut/session/event/SessionCreatedEvent.html[SessionCreatedEvent]
-|Fired when a link:{micronautsessionapi}/io/micronaut/session/Session.html[Session] is created
-
-|link:{micronautsessionapi}/io/micronaut/session/event/SessionDeletedEvent.html[SessionDeletedEvent]
-|Fired when a link:{micronautsessionapi}/io/micronaut/session/Session.html[Session] is deleted
-
-|link:{micronautsessionapi}/io/micronaut/session/event/SessionExpiredEvent.html[SessionExpiredEvent]
-|Fired when a link:{micronautsessionapi}/io/micronaut/session/Session.html[Session] expires
-
-|link:{micronautsessionapi}/io/micronaut/session/event/SessionDestroyedEvent.html[SessionDestroyedEvent]
-|Parent of both `SessionDeletedEvent` and `SessionExpiredEvent`
-
-|===
+See the documentation for link:https://micronaut-projects.github.io/micronaut-session/snapshot/guide/[Micronaut Session] for information about supporting HTTP sessions in your applications.
```

Comment:
the url should be defined in `gradle.properties`

Reply:
Done via https://github.com/micronaut-projects/micronaut-core/pull/8737/commits/fb1a89dd26cce7acc4ab4622fae2a432daeaa431